Source/Events.Processing/MissingEventInformation.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Dolittle.SDK.Events.Processing
+{
+    /// <summary>
+    /// Exception that gets thrown when there are details missing on an event.
+    /// </summary>
+    public class MissingEventInformation : Exception
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="MissingEventInformation"/> class.
+        /// </summary>
+        /// <param name="details">The details on what is missing.</param>
+        public MissingEventInformation(string details)
+            : base($"Missing information on event: {details}")
```

Comment:
```suggestion
            : base($"Event is missing information about its {details}")
```
Flows a bit nicer, when we have exceptions like these where the details are on a string I think it's nice that it flows nicely, kinda like how our tests also have the BDD-style descriptions to them.